moped-editor/src/views/projects/projectView/FundingDeptUnitAutocomplete.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { CircularProgress, TextField } from "@material-ui/core";
+import { Autocomplete, Alert } from "@material-ui/lab";
+import { useSocrataJson } from "src/utils/socrataHelpers";
+import { filterOptions } from "src/utils/autocompleteHelpers";
+
+/*
+* Transportation Project Financial Codes
+*/
+const SOCRATA_ENDPOINT =
+  "https://data.austintexas.gov/resource/bgrt-2m2z.json?dept_unit_status=Active&$limit=9999";
+
+const DeptUnitInput = (params, error = false, variant) => {
+  return (
+    <TextField
+      {...params}
+      error={error}
+      InputLabelProps={{ required: false }}
+      label="Dept-unit"
+      variant={variant}
+    />
+  );
+};
+
+/**
+ * Material Autocomplete wrapper that enables selecting a dept unit record from a
+ * Socrata dataset and setting it.
+ * * @param {Object} classes - MaterialUI style object
+ * * @param {Object} props - passes down props parent MUI Table
+ * * @param {Array} value - passes an array of objects, each representing a dept unit
+ *  @return {JSX.Element}
+ */
+const FundingDeptUnitAutocomplete = ({ classes, props, value }) => {
+  const { data, loading, error } = useSocrataJson(SOCRATA_ENDPOINT);
+  console.log(data)
+  console.log(classes)
```

Comment:
Would you consider changing these to `console.info()`'s or removing them?

Reply:
i forgot to remove them! thanks for catching it